sys/net/application_layer/cord/doc.txt

```diff
@@ -49,15 +49,14 @@
  * # Structure
  *
  * This module is structured in a number of submodules with goal to reflect the
- * different roles described in `draft-ietf-core-resource-directory-27`:
+ * different roles described in `RFC 9176`:
  *
  * - `cord_ep`:     standard endpoint implementation following the rules as
  *                  defined i.a. in sections 5.2, 5.3, A.1, and A.2
  * - `cord_epsim`:  endpoint implementation following the simple registration
  *                  procedure as defined in section 5.3.1
  * - `cord_lc`:     lookup client implementation for querying information from
- *                  an RD using the lookup and group interfaces (**NOT
- *                  YET IMPLEMENTED**)
+ *                  an RD using the lookup and group interfaces
```

Comment:
I understood that not-yet-implemented to refer only to the group part. The fact that grepping for rd-group comes up empty indicates that group lookup is not implemented.

However, with how groups turned out to be done in RD, it may not be necessary to do anything extra, so maybe

```suggestion
 *                  an RD using the lookup interfaces
```

(still plural, as there is resource and registration lookup)

Reply:
I understand your point. The main reason for removing the label was that **Not Yet Implemented** sounded to me like the whole `cord_lc` was not implemented yet. I thought while there may still be limitations (like the group part), the `cord_lc` functions for the normal lookup seemed to be implemented. In this case I approve of the suggested change.